### DISCUSSION TITLE
How to redirect to different hosts/ports based on url path?

### DISCUSSION THREAD

**Discussion by europacafe** [NginxProxyManager/nginx-proxy-manager#3524] [General]
Example:
www.example.com ----> 192.168.1.1:81
www.example.com/weather ----> 192.168.1.1:82

How could I accomplish the above examples?...thanks.

**Comment by CodeWithCJ** [comment]
Did you find out the answer? I am also stuck on the same and also unable to access via 443 port

**Comment by DaKa22** [comment]
Esto te puede ayudar, necesitar usar el apartado de custom locations
![Captura de pantalla 2024-02-22 090529](https://github.com/NginxProxyManager/nginx-proxy-manager/assets/55817715/091af21d-d696-4e25-9759-e0650c9a9bc6)
![image](https://github.com/NginxProxyManager/nginx-proxy-manager/assets/55817715/708b163f-7e94-46c3-b431-ac4a7268cb61)

**Reply by malavolti** [reply]
@DaKa22 I've tried what you suggested, but it doesn't work in my case.

I have spawn with Docker the NPM on my server and I want to public different services through other docker containers on other ports.

I've opened ports 80, 81, 443 for NPM and I have connected the server running the NPM container as a Proxy Host like this:

**Domain Name:** `www.example.com`
**Scheme:** `http`
**Forward Hostname/IP:** `192.168.1.1`
**Forward Port:** `81`

Then I have added a custom Location "**/phpmyadmin**" for the Docker container that works on the port 8081 with:
**Location:** `/phpmyadmin`
**Scheme:** `http`
**Forward Port:** `8081`

But NPM stop itself with the error:
the "user" directive makes sense only if the master process runs with super-user privileges, ignored in /etc/nginx/nginx.conf:4

Docker containers:
``` bash
PORTS                                                                                  NAMES
0.0.0.0:80-81->80-81/tcp, :::80-81->80-81/tcp, 0.0.0.0:443->443/tcp, :::443->443/tcp   proxy-https
0.0.0.0:8081->80/tcp, :::8081->80/tcp                                                  phpmyadmin
```

Can someone help me, please?

Thank you so much!
Marco

**Comment by SeanChengN** [comment]
Hi, any progress?

**Comment by wagnbeu0** [comment]
Hi, I am also using the Manager now, but something is not working as expected.

I have a host virtual.homenet.org, and I want that all incoming requests are route to http://192.168.1.10:80/application

How can I do this?

**Comment by brainroot** [comment]
I resove this (for me) 
![Screenshot_20250301_162036](https://github.com/user-attachments/assets/8c1c7834-8279-4ada-b11d-41f9e5e89c89)
Set server ip for location.
If need you can set port.

**Reply by brosahay** [reply]
can you kindly share the other details? because I tried a few combinations and it doesn't work for me.
